README.md

```diff
@@ -172,33 +176,37 @@ you would pandas.
 
 <img align="right" style="display:inline;" height="350" width="300" src="https://github.com/modin-project/modin/blob/master/docs/img/read_csv_benchmark.png?raw=true"></a>
 
-The `modin.pandas` DataFrame is an extremely light-weight parallel DataFrame. Modin 
-transparently distributes the data and computation so that all you need to do is
-continue using the pandas API as you were before installing Modin. Unlike other parallel
-DataFrame systems, Modin is an extremely light-weight, robust DataFrame. Because it is
-so light-weight, Modin provides speed-ups of up to 4x on a laptop with 4 physical cores.
+The `modin.pandas` DataFrame is an extremely light-weight parallel DataFrame. To see how Modin compares
+to other dataframe systems (namely Dask DataFrame and Koalas), check out
```

Comment:
nit: delete the sentence "To see how Modin compares..." since it's included in our FAQs, or move it to after the "... while working with more data faster." sentence for better flow.